lib/dataset_augmenter.js

```diff
@@ -34,6 +34,7 @@ module.exports = class DatasetAugmenter extends Stream.Transform {
         this._schemas = schemaRetriever;
         this._paramReplacer = new ParameterReplacer(this._schemas, this._constProvider, {
             locale: options.locale,
+            paramLocale: options.paramLocale,
```

Comment:
What's this for?

Reply:
This is for when you wanna use a different language for parameters.
For translation task for example, I replace params with Farsi before translation so sentence is English but param is Farsi.